app/src/main/java/com/example/sharewhatyoucanproject/review/ReviewFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.sharewhatyoucanproject.review
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.sharewhatyoucanproject.R
+import com.example.sharewhatyoucanproject.databinding.FragmentReviewBinding
+
+class ReviewFragment : Fragment() {
+    private var _binding: FragmentReviewBinding? = null
+    private val binding get() = _binding!!
```

Comment:
make sure to null out the binding in `onDestroyView`

Reply:
done